### ISSUE TITLE
Photos Not Appearing

### ISSUE THREAD

**Issue by dc-bond** [julianpoy/RecipeSage-selfhost#37]
Same issue as #29 - I'm running what appears to be the latest version (2.9.9) on a linux host but the recipe photos don't appear. Everything else seems to work great.

I * think * a relevant log output is:

`2023-01-04T01:20:16.571167295Z 2023/01/04 01:20:16 [warn] 28#28: *208 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000004, client: 192.168.1.2, server: localhost, request: "POST /api/images?token=1a5a7e4d812a3709e2be5b5e5789a35d2565f274feb97824f9f113b27678a0401ebb6683c02a9b37e50371551e29fca8& HTTP/1.1", host: "192.168.1.2:46789", referrer: "http://192.168.1.2:46789/"
2023-01-04T01:20:16.738510986Z 192.168.1.2 - - [04/Jan/2023:01:20:16 +0000] "POST /api/images?token=1a5a7e4d812a3709e2be5b5e5789a35d2565f274feb97824f9f113b27678a0401ebb6683c02a9b37e50371551e29fca8& HTTP/1.1" 200 456 "http://192.168.1.2:46789/" "Mozilla/5.0 (X11; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0" "-"
2023-01-04T01:20:16.842654900Z 192.168.1.2 - - [04/Jan/2023:01:20:16 +0000] "GET /minio/recipesage-selfhost/1672795216621 HTTP/1.1" 403 327 "http://192.168.1.2:46789/" "Mozilla/5.0 (X11; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0" "-"`

**Comment by julianpoy** [comment]
Just checked out the latest from this repository and can't reproduce the issue you're seeing. Can you post the logs from:
```
docker-compose logs create-minio-buckets
```
and
```
docker-compose logs minio
```

**Comment by dc-bond** [comment]
So I can show you logs if still desired but I just got it working. I'm not completely sure why this fixed my issue, but your canonical docker-compose on line 35 uses `/minio/recipesage-selfhost/` as the env variable and also sets `myminio/recipesage-selfhost` on line 115. Your proxy.conf sets `location /minio/ {` on line 20. 

When I changed the env variable on line 35 to `/myminio/recipesage-selfhost/` and also changed proxy.conf line 20 to `location /myminio/ {` - the photos started working.
